Makefile

```diff
@@ -69,21 +70,26 @@ clean:
 	rm -rf ansible-runner.egg-info
 	rm -rf rpm-build
 	rm -rf deb-build
+	rm -f setup.py
 	find . -type f -regex ".*\py[co]$$" -delete
 
 dist:
-	$(DIST_PYTHON) setup.py bdist_wheel --universal
+	poetry build
 
 sdist: dist/$(NAME)-$(VERSION).tar.gz
 
+# Generate setup.py transiently for the sdist so we don't have to deal with
+# packaging poetry as a RPM for rpm build time dependencies.
 dist/$(NAME)-$(VERSION).tar.gz:
-	$(DIST_PYTHON) setup.py sdist
+	$(PYTHON) packaging/poetry-gen-setup.py
+	poetry build -f sdist
+	rm -f setup.py
```

Comment:
Should we drop this line now that it's checked into the repo?

Reply:
@shanemcd yes, good catch. Thanks :)